include/Global.h

```diff
@@ -290,6 +290,7 @@ extern char       (*h_DE_Array_G      [2])[PS1][PS1][PS1];
 extern real       (*h_Emag_Array_G    [2])[PS1][PS1][PS1];
 #endif
 extern real        *h_ExtPotTable;
+extern void       **h_GenePtrArray;
```

Comment:
Would you mind renaming it as `h_ExtPotGenePtr`? It will improve consistency since all external-potential variables share the same keyword `ExtPot`. If you agree, apply it throughout this PR.